tests/monotouch-test/NearbyInteraction/NINearbyObjectTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2020 Microsoft Corp.
+
+#if !MONOMAC
+
+using System;
+using Foundation;
+using ObjCRuntime;
+using NearbyInteraction;
+using NUnit.Framework;
+using OpenTK;
+
+namespace MonoTouchFixtures.NearbyInteraction {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NINearbyObjectTest {
+
+		[Test]
+		public void DirectionNotAvailable ()
+		{
+			if (!TestRuntime.CheckXcodeVersion (12, 0))
+				Assert.Ignore ("Requires iOS 14.0");
```

Comment:
Really minor, so no need to fix this, but for the future this is shorter:

```suggestion
			TestRuntime.AssertXcodeVersion (12, 0);
```